x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/esql/action/ColumnInfo.java

```diff
@@ -25,5 +25,5 @@ static ColumnInfo fromXContent(XContentParser parser) {
 
     String name();
 
-    String esType();
+    String outputType();
```

Comment:
++, this name makes more sense.

Reply:
It's a little frustrating to me that those have subtly different behavior, but this is in fact the one we need here.